valohai/internals/utils.py

```diff
@@ -0,0 +1,2 @@
+def uri_to_filename(uri: str) -> str:
```

Comment:
Should this take care of query parameters too? If my URL is `https://example.com/baz/quux.png?ts=9310101`...